src/ucp/wireup/wireup_cm.c

```diff
@@ -1374,23 +1401,14 @@ ucp_request_t* ucp_ep_cm_close_request_get(ucp_ep_h ep, const ucp_request_param_
 
 static int ucp_cm_cbs_remove_filter(const ucs_callbackq_elem_t *elem, void *arg)
 {
-    ucp_cm_client_connect_progress_arg_t *client_connect_arg;
-
-    if (elem->cb == ucp_cm_client_connect_progress) {
-        client_connect_arg = elem->arg;
-        if (client_connect_arg->ucp_ep == arg) {
-            ucp_cm_client_connect_prog_arg_free(client_connect_arg);
-            return 1;
-        } else {
-            return 0;
-        }
-    } else if ((elem->cb == ucp_ep_cm_disconnect_progress)      ||
-               (elem->cb == ucp_cm_server_conn_notify_progress) ||
+    if (ucp_cm_connect_progress_remove_filter(elem, arg)) {
```

Comment:
why do we also need to remove callbacks here? isn't it enough to remove them in ucp_ep_cm_disconnect_cm_lane?

Reply:
no need anymore